site-src/concepts/glossary.md

```diff
@@ -0,0 +1,66 @@
+# Gateway API Glossary
+
+### Consumer Route
+
+A Route bound to a workload's Service by a consumer of a given workload,
+refining the specific consumer's use of the workload.
+
+### Gateway Controller
+
+A _gateway controller_ is software that runs on a cluster to manage the
+infrastructure associated with routing traffic from outside the cluster to
+inside the cluster (north/south traffic), analogous to the earlier _ingress
```

Comment:
Same comment as above, need to be careful to avoid limiting the scope of Gateways here.

Reply:
Does 5783333f78307a4a037b61c111788454659a692b look good for this?